.flowconfig

```diff
@@ -20,6 +20,9 @@ ui/src/__flow__/types.js
 ui/src/__flow__/preact.js
 ui/src/__flow__/emotion.js
 
+####### ATOM LIBRARY #######
+node_modules/@guardian/atom-renderer/dist/__flow__/types
```

Comment:
@sndrs I'm hoping this is a fair compromise.

Reply:
can you explain again why the `file.js.flow` thing didn't work? (sorry!)

Reply:
done offline